src/main/java/servlets/VendorServlet.java

```diff
@@ -33,12 +45,73 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     JsonConverter converter = new JsonConverter();
     String configMap = converter.getConfigMap();
-    response.setContentType("text/html;");
+    response.setContentType(CONTENT_TYPE);
     response.getWriter().println(configMap);
   }
 
-  @Override
-  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // DELETE method --> deletes vendor configuration directory
+  /** Delete the desired Vendor from the fileset and spreadsheets.*/
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    String vendorID = request.getParameter(VENDOR_ID_PARAM);
+    String responseString = "success"; 
+
+    // Delete the file from the filesystem and update the sheets.
+    try {
+      deleteVendorFile(vendorID);
+      updateSheets(request);    
+    } catch (IOException e) {
+      responseString = "Failed to Delete File.";
+    } catch (GeneralSecurityException e) {
+      responseString = "Failed to Delete File.";
+    } 
+
+    // Convert the String message into a JSON String.
+    String jsonMessage = messageAsJson(responseString);
+
+    response.setContentType(CONTENT_TYPE);
```

Comment:
The Content Type probably needs to be application/json since we are returning JSON.

I think Vincent had trouble with text/html and had to change that to JSON in a few places.

Reply:
Got it fixed.